jipdate.py

```diff
@@ -229,16 +259,28 @@ def get_jira_issues(jira, username):
 
     f.write(msg)
     vprint("Found issue:")
+
+    unsorted_issues={}
     for issue in my_issues:
+        issue_id = "%s" % issue
+        this_issue = {}
+
+        this_issue["comments"] = ""
         vprint("%s : %s" % (issue, issue.fields.summary))
-        f.write("[%s]\n" % issue)
-        f.write("# Header: %s\n" % issue.fields.summary)
-        f.write("# Type: %s\n" % issue.fields.issuetype)
-        f.write("# Status: %s\n" % issue.fields.status)
-        f.write(get_extra_comments())
-        if last_comment:
-            write_last_jira_comment(f, jira, issue)
-        f.write("\n")
+        this_issue["comments"] += "[%s]\n" % issue
```

Comment:
I see, that's a bit different to how I first understood that it was supposed to be used. I was thinking it was just going to add a parent link to each issue. Let me think a bit more about this.